packages/gensx/src/checkpoint.ts

```diff
@@ -194,22 +216,34 @@ export class CheckpointManager implements CheckpointWriter {
     if (!this.root) return;
 
     try {
-      const baseUrl =
-        process.env.GENSX_CHECKPOINT_URL ?? "http://localhost:3000";
-      const url = join(baseUrl, "api/execution");
-
       // Create a deep copy of the execution tree for masking
       const maskedRoot = this.maskExecutionTree(structuredClone(this.root));
-
+      const baseUrl =
+        process.env.GENSX_CHECKPOINT_URL ?? "http://localhost:3001";
```

Comment:
Very shortly we should change this default to be the production API. If this actually works in production, maybe we should change it now?

Reply:
Sure I can update it.